projectorganizer/src/prjorg-project.c

```diff
@@ -497,13 +509,51 @@ gchar **prjorg_project_load_expanded_paths(GKeyFile * key_file)
 }
 
 
+static GPtrArray *get_session_files(GKeyFile *config)
```

Comment:
This is mostly ~~stolen from~~ inspired by `configuration_load_session_files` in Geany. The only difference is that this one only reads the filename.